### PR TITLE
Add prombolt to third-party utilities

### PR DESCRIPTION
**Pull request by mdlayher**
Adds my [Bolt](https://github.com/boltdb/bolt) collector package to other third party utilities.

### CODE REVIEW

**Comment by juliusv** [comment]
Thanks! I just merged https://github.com/prometheus/docs/pull/578, which introduces categories to the exporter list. Could you rebase ontop of the latest master?

**Comment by mdlayher** [comment]
@juliusv updated.  I think this still technically qualifies as a "third party utility" since it's a package you add to your applications to enable bolt instrumentation.  Let me know if you think otherwise!

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ practices.
    * Clojure: [prometheus-clj](https://github.com/soundcloud/prometheus-clj)
    * Go: [go-metrics instrumentation library](https://github.com/armon/go-metrics)
    * Go: [gokit](https://github.com/peterbourgon/gokit)
+   * Go: [prombolt](https://github.com/mdlayher/prombolt)
```
Comment (grobie) [inline content/docs/instrumenting/exporters.md]: This should go to the Databases section I believe, as it is an exporter for bolt?

**Comment by brian-brazil** [comment]
Thanks!

File: content/docs/instrumenting/exporters.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: It appears to be a library rather than an exporter.

**Pull request merged**